src/mappening/api/locations.py

```diff
@@ -0,0 +1,154 @@
+# TODO MAJOR CLEANUP but I'm lazy
+from mappening.utils.database import *
+from mappening.api.utils import location_utils, tokenize
+
+from flask import Flask, jsonify, request, json, Blueprint
+from flask_cors import CORS, cross_origin
+import re
+import os
+from operator import itemgetter
+
+# Must be on same level as app.py
+# data = json.load(open('sampleData.json'))
+
+locations = Blueprint('locations', __name__)
+
+# Returns JSON of all past locations/venues
+@locations.route('/', methods=['GET'])
+def get_all_locations():
+    output = []
+
+    locations_cursor = UCLA_locations_collection.find({}, {'_id': False})
+    if locations_cursor.count() > 0:
+      for loc in locations_cursor:
+        output.append({"location": loc})
+    else:
+        print 'Cannot find any locations!'
+
+    # Output typically contains name, city, country, latitude, longitude, state, 
+    # street, and zip for each location
+    return jsonify({'locations': output})
+
+# UPDATE DATABASE
+
+# Add locations to database from given collection
+# Sample collection(s): events_ml_collection, ucla_events_collection
+# TODO: hook up so everytime we get new events we add their location data to db
+@locations.route('/add/<events_collection>', methods=['PUT'])
```

Comment:
Why is this a route rather than a function you can call? Seems dangerous to make this API callable

Reply:
Ah I actually meant to do that but I was using it for testing at the time.